jit-compiler/src/lib.rs

```diff
@@ -53,7 +53,8 @@ pub fn compile<T: FieldElement>(
     analyzed: &Analyzed<T>,
     requested_symbols: &[&str],
 ) -> Result<CompiledPIL, String> {
-    log::info!("JIT-compiling {} symbols...", requested_symbols.len());
+    // TODO this should be changed back to Info after the introduction of the ToCol trait.
+    log::debug!("JIT-compiling {} symbols...", requested_symbols.len());
```

Comment:
changed this one as well for context

Reply:
This is not related to the ToCol trait.

Reply:
Yea but it gets implied from the other ones